Lib/argparse.py

```diff
@@ -688,6 +747,85 @@ def _get_default_metavar_for_positional(self, action):
         return action.type.__name__
 
 
+class _ManpageFormatter(HelpFormatter):
+    def _markup(self, text):
+        return text.replace('-', '\\-')
+
+    def _underline(self, text):
+        return '\\fI\\,{}\\/\\fR'.format(text)
+
+    def _bold(self, text):
+        if not text.strip().startswith('\\fB'):
+            text = '\\fB{}'.format(text)
+        if not text.strip().endswith('\\fR'):
+            text = '{}\\fR'.format(text)
+        return text
+
+    def _format_action_invocation(self, action):
+        if not action.option_strings:
+            metavar, = self._metavar_formatter(action, action.dest)(1)
+            return metavar
+        else:
+            parts = []
+
+            # if the Optional doesn't take a value, format is:
+            #    -s, --long
+            if action.nargs == 0:
+                parts.extend(map(self._bold, action.option_strings))
+
+            # if the Optional takes a value, format is:
+            #    -s ARGS, --long ARGS
+            else:
+                default = self._underline(action.dest.upper())
+                args_string = self._format_args(action, default)
+                for option_string in action.option_strings:
+                    parts.append('{} {}'.format(self._bold(option_string),
+                                                args_string))
+            return ', '.join(parts)
+
+    def _format_action(self, action):
+        # determine the required width and the entry label
+        help_position = min(self._action_max_length + 2,
+                            self._max_help_position)
+        action_width = help_position - self._current_indent - 2
+        action_header = self._format_action_invocation(action)
+
+        # no help; start on same line and add a final newline
+        if not action.help:
+            tup = self._current_indent, '', action_header
+            action_header = '%*s%s' % tup
+
+        # short action name; start on the same line and pad two spaces
+        elif len(action_header) <= action_width:
+            tup = self._current_indent, '', action_width, action_header
+            action_header = '%*s%-*s  ' % tup
```

Comment:
Not sure if this is better or not, letting this be a longer line.

This code is from `argparse` itself.